src/ray/raylet/node_manager.cc

```diff
@@ -41,6 +41,14 @@ RAY_CHECK_ENUM(protocol::MessageType::GetActorFrontierReply,
 RAY_CHECK_ENUM(protocol::MessageType::SetActorFrontier,
                local_scheduler_protocol::MessageType::SetActorFrontier);
 
+// TODO(rkn): This function appears in at least three places. It should be deduplicated.
+int64_t current_time_ms_duplicate() {
```

Comment:
Let's put this one into `src/ray/util/util.h` (without the _duplicate) then so we can start sorting the other ones out then.